laws/src/main/scala/cats/laws/MonoidalLaws.scala

```diff
@@ -0,0 +1,16 @@
+package cats.laws
+
+import cats.Monoidal
+
+trait MonoidalLaws[F[_]] {
```

Comment:
We'd then bind these laws in `InvariantTests`, using the `Invariant[F]` instance.